azure/packages/azure-client/src/index.ts

```diff
@@ -28,5 +28,6 @@ export {
 	ITelemetryBaseLogger,
 } from "./interfaces";
 
+export { ContainerErrorType } from "@fluidframework/container-definitions";
```

Comment:
We'll want to add this export to the custom data rollup logic in `rollup-api-data.js` too, so the docs show up on the website (until we come up with a better long-term fix for the doc generation).